### PR TITLE
Add lobby checkbox instead of radio buttons for enabling lobby.

### LINKED ISSUE #244: Error message when both video&audio don’t work (like Firefox non-HTTPS)
When neither your video&audio work, it’s really not necessary to show the placeholder and the icons in the bottom right. They only distract from the error message in the center.

Also then, there should be a more specific error message in the main view. Like when you use Firefox and your instance does not have HTTPS it currently says:
> Error while accessing microphone & camera
> Starting video failed

Better would be:

> Calling only works over a secure connection
> Ask your admin to enable https for your Nextcloud

cc @nextcloud/spreed

### LINKED ISSUE #2244: Make Lobby feature easier discoverable
I just tried out the lobby feature of Nextcloud Talk 7.0 and it works really nice! :slightly_smiling_face: 

But the menu was quite confusing for me:

image

"Enable for" but what? What is the affect for the user? Also if I click on "moderators only" the drop down closes immediately which means that almost noone will discover that they can add a start data.

What about having a check box instead:

````[ ] Lobby````

Maybe with a tooltip which describes it in a short sentences like "non-moderators have to wait in a Lobby until the call starts". If you check it the dialog should stay open and the input field for the start date should be shown so that the user actually sees that there are further config options for the meeting.

What do you think?

cc @nextcloud/designers

### PR DESCRIPTION
**Pull request by Ivansss**
Fix #2244

Screenshot 2019-10-04 at 15 36 37

### CODE REVIEW

File: js/views/templates/callinfoview.handlebars [hunk 1]
```diff
@@ -44,22 +44,16 @@
 				</li>
 				<li>
 					<span class="menuitem caption">
-						<span>{{enableForLabel}}</span>
+						<span>{{webinarLabel}}</span>
```
Comment (nickvergessen) [inline js/views/templates/callinfoview.handlebars]:
```suggestion
						<span>{{webinaryLabel}}</span>
```

**Comment by nickvergessen** [comment]
To be inline with the guest checkbox above and the input of the password, we should have an icon with a clock or the lobby bell on the starttime thingy?

**Comment by Ivansss** [comment]
Yes, I was going to open an issue about that.
There is already an `icon-calendar` next to the "start time" input, but it's not visible.
It is visible when changed to `icon-calendar-dark` but it is not aligned with the input field.

File: js/views/callinfoview.js [hunk 2]
```diff
@@ -55,13 +56,9 @@
...
 				copyLinkLabel: t('spreed', 'Copy link'),
-				enableForLabel: t('spreed', 'Enable for'),
-				allParticipantsLabel: t('spreed', 'All participants'),
-				moderatorsOnlyLabel: t('spreed', 'Moderators only'),
-				allParticipantsValue: OCA.SpreedMe.app.LOBBY_NONE,
-				moderatorsOnlyValue: OCA.SpreedMe.app.LOBBY_NON_MODERATORS,
-				lobbyStateAllParticipants: this.model.get('lobbyState') === OCA.SpreedMe.app.LOBBY_NONE,
-				lobbyStateModeratorsOnly: this.model.get('lobbyState') === OCA.SpreedMe.app.LOBBY_NON_MODERATORS,
+				webinarLabel: t('spreed', 'Webinar'),
+				lobbyCheckboxLabel: t('spreed', 'Use lobby'),
```
Comment (nickvergessen) [inline js/views/callinfoview.js]:
```suggestion
				lobbyCheckboxLabel: t('spreed', 'Activate lobby'),
```

File: js/views/callinfoview.js [hunk 3]
```diff
@@ -203,6 +195,12 @@
 			var roomURL = OC.generateUrl('/call/' + this.model.get('token')),
 				completeURL = window.location.protocol + '//' + window.location.host + roomURL;
 
+			this.ui.lobbyCheckboxLabel.tooltip({
+				placement: 'bottom',
+				trigger: 'hover',
+				title: t('spreed', 'Only moderators can enter the room until the meeting is started for everyone.')
```
Comment (nickvergessen) [inline js/views/callinfoview.js]:
```suggestion
				title: t('spreed', 'Only moderators can enter the conversation until the meeting is started for everyone.')
```

Comment (nickvergessen) [reply js/views/callinfoview.js]: or even meeting?

**Comment by danxuliu** [comment]
Squashed, rebased, fixed an issue with dangling tooltips, replaced the tooltip for _Enable lobby_ with a description embedded in the menu itself, and fixed the wording as agreed on Talk:

Talk-Moderation-Menu-Enable-Lobby

**Comment by jancborchardt** [comment]
Just a small thing → can we change it from "Webinar" to the more common and slightly more professional sounding "Web conference"? :D

**Comment by nickvergessen** [comment]
Let's do that once we have all the "professional" feature expect from a real conference?

**Pull request merged**